conftest.py

```diff
@@ -617,18 +616,11 @@ def _update_os_related_config():
         log_file=tests_log_file,
         log_level=session.config.getoption("log_cli_level") or logging.INFO,
     )
-    py_config_scs = py_config.get("storage_class_matrix", [])
-
-    # Set py_config["storage_class_matrix"]
-    if not skip_if_pytest_flags_exists(pytest_config=session.config):
-        cluster_storage_classes_names = [sc.name for sc in list(StorageClass.get(dyn_client=get_client()))]
-        # If TOPOLVM storage class is present in the cluster - add it to the matrix
-        # And do not add the HPP storage classes
-        if StorageClassNames.TOPOLVM in cluster_storage_classes_names:
-            py_config_scs.extend(py_config["topolvm_storage_class_matrix"])
-        # If HPP CSI storage class is present - add HPP CSI storage classes
-        elif HppCsiStorageClass.Name.HOSTPATH_CSI_BASIC in cluster_storage_classes_names:
-            py_config_scs.extend(py_config["new_hpp_storage_class_matrix"])
+    # Add HPP-CSI-BASIC/HPP-CSI-PVC-BLOCK to global config's storage_class_matrix, only
```

Comment:
Could you please keep this comment?
`# Set py_config["storage_class_matrix"]`

Reply:
But this comment would be wrong. We are not setting `py_config["storage_class_matrix"]`

Reply:
Is there a better wording to explain what are we doing here with the `py_config["storage_class_matrix"]`?

Reply:
I will update the readme file to add this info.